src/app/tenant/pages/tenant-search/tenant-search.component.ts

```diff
@@ -22,7 +23,11 @@ import { TenantSearchViewModel } from './tenant-search.viewmodel'
   styleUrls: ['./tenant-search.component.scss']
 })
 export class TenantSearchComponent implements OnInit {
-  viewModel$: Observable<TenantSearchViewModel> = this.store.select(selectTenantSearchViewModel)
+  viewModel$: Observable<TenantSearchViewModel> = this.store.select(selectTenantSearchViewModel).pipe(
+    tap((vm) => {
```

Comment:
I would do it with a subscribe to this.viewModel$ in ngOnInit because you solution is only working as long as someone else is subscribing to the observable